eng/common/scripts/link-replacement.ps1

```diff
@@ -0,0 +1,54 @@
+param (
+  # url list to verify links. Can either be a http address or a local file request. Local file paths support md and html files.
+  [string] $sourceDir,
+  # file that contains a set of links to ignore when verifying
+  [string] $ignoreLinksFile = "$PSScriptRoot/ignore-links.txt",
```

Comment:
Doesn't look like this is used anywhere.

Reply:
I added into copy doc to blobstorage ps. 
The idea currently serve as artiface-metadata-parsing. It is not used in any template.